splunk_add_on_ucc_framework/global_config.py

```diff
@@ -79,13 +79,23 @@ def from_file(cls, global_config_path: str) -> "GlobalConfig":
         )
         return GlobalConfig(content, is_global_config_yaml)
 
+    def __eq__(self, other: object) -> bool:
```

Comment:
Are we planning any use case to compare two GlobalConfig objects?

Reply:
Yes, in the next PR, there is already a discussion there, so closing this one.